inc/commondbtm.class.php

```diff
@@ -5236,7 +5236,7 @@ function addFiles(array $input, $options = []) {
             }
 
             $input2["entities_id"]             = $entities_id;
-            $input2["is_recursive"]            = 1;
+            $input2["is_recursive"]            = 0;
```

Comment:
I think `is_recursive` should have a value based on source item `is_recursive` property.

Reply:
Agree with that 
Note that this is only used by

- ```CommonItilObject```
- ```ItilFollowup```
- ```ItilSolution```
- ```CommonItilTask```
- ```KnowBaseItem```

that does not contain the ```is_recusrive``` key (maybe in future)

plugins could eventually use it